hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/http/AbstractHttpFileSystem.java

```diff
@@ -39,13 +40,19 @@
 abstract class AbstractHttpFileSystem extends FileSystem {
   private static final long DEFAULT_BLOCK_SIZE = 4096;
   private static final Path WORKING_DIR = new Path("/");
+  private static final String CONF_HTTP_READ_TIMEOUT = "yarn.nodemanager.localizer.http.read.timeout.ms";
```

Comment:
1. these should be somewhere where they can be referred to in code, such as a new class `HttpFileSystemConstants`
2. And their name must be that of the filesystem class, not anything related to the RM.